src/universal/components/EditorLinkChanger/EditorLinkChanger.js

```diff
@@ -46,6 +46,16 @@ class EditorLinkChanger extends Component {
     }
   };
 
+  componentWillMount() {
+    const {trackEditingClient} = this.props;
+    trackEditingClient('editor-link-changer', true);
```

Comment:
Just making a note here that static IDs work perfectly fine for the current use case, but in a world where there can be multiple modals, we'd want to use proper unique IDs.